packages/openapi-fetch/package.json

```
@@ -9,6 +9,15 @@
   "license": "MIT",
   "type": "module",
   "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
```

```suggestion
    },
    "*": "./*"
```

we’ll want to expose all files in this package like before to avoid breaking changes. by default, package imports make all non-explicit imports private